docs/UserGuide.md

```diff
@@ -1,42 +1,259 @@
+<div style="page-break-after: always; visibility: hidden"> 
+\pagebreak 
+</div>
+
+## Quick start
+
 
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+
+2. Download the latest "planus.jar" from [here](https://github.com/AY2021S1-CS2113T-W12-1/tp/releases/download/v1.0/planus.jar).
+
+3. For first time Windows user, open command prompt and run the following line to enable color display of command prompt:
+
+    `reg add HKCU\Console /v VirtualTerminalLevel /t REG_DWORD /d 1`
+    
+4. Open a **new** command prompt/terminal and run "planus.jar" with following command:
+
+    `java -jar planus.jar`
+    
+    Following content is display if there's no existing data file:
+    
+    ![initial display](images/init.PNG)
+    
+    
+5. **Maximise** the command prompt/terminal window to get a full display of the contents.
+
+6. Type a command in command prompt/terminal and press Enter to execute it.
+
+7. Refer to the [Features](#features) below for details of each command.
+
+
+<div style="page-break-after: always; visibility: hidden"> 
+\pagebreak 
+</div>
+
 
 ## Features 
 
-{Give detailed description of each feature}
 
-### Adding a todo: `todo`
-Adds a new item to the list of todo items.
+### Notes about command format:
+```
+- Words in UPPER_CASE are the parameters to be supplied by the user. 
+    - e.g. in "add DESCRIPTION [d/DATE] [st/START_TIME] [et/END_TIME] [p/PRIORITY]", 
+    DESCRIPTION is a parameter which can be used as "add user guide".
+      
+- Parameter in square bracket is optional. 
+    - e.g. in "add DESCRIPTION [d/DATE] [st/START_TIME] [et/END_TIME] [p/PRIORITY]", 
+    DATE, START_TIME, END_TIME and PRIORITY are optional parameters which can be omitted.
+    
+- Within square bracket, optional flags could be enabled, flags are separated by '|'.
+    - e.g. in "list [-d|-p|-w|-m]",
+    The flag parameter is an optional parameter. When it is used, any one of the flags
+    like -d or -p or -w or -m could be enabled. 
+    
+- Input date format is dd-MM-yyyy.
+    - e.g. 20/02/2020
+      
+- Input time format is HHmm in 24-hrs.
+    - e.g. 1830
+    
+- Input priority format is 1, 2 or 3 represents LOW, MEDIUM and HIGH respectively.
+    - e.g. 1
 
-Format: `todo n/TODO_NAME d/DEADLINE`
+- Index of a task is the number displayed after "#" and before description of the task.
+    - e.g. "#1029 meeting", 1029 is the index.
+```    
+    
+### <span style="color:red">Showing all commands</span> : `help`
 
-* The `DEADLINE` can be in a natural language format.
-* The `TODO_NAME` cannot contain punctuation.  
+Shows the commands available with the formats.
+
+Format: `help`
 
 Example of usage: 
 
-`todo n/Write the rest of the User Guide d/next week`
+`help`
+
+Expected outcome:
+
+![help](images/help.PNG)
+
+### <span style="color:red">Adding a task</span> : `add`
+
+Add a task to the task list and display a message.
```

Comment:
Perhaps, you can add sequence diagrams to the methods and features that you mentioned. It would certainly add more value to each of the features.